### ISSUE TITLE
Cleanup coding standards

### ISSUE THREAD

**Issue by AlexSkrypnyk** [salsadigitalauorg/merlin-framework#161]
The project uses mixed coding standards:
- some files have an indentation of 2 spaces, some 4
- vertical spacing is not followed consistently
- comments formatting is not followed consistently
- CI does not not scan `tests` dir

Is it possible to utilise Drupal or Symfony2 standard for this project (having a completely custom standard with 49 rules is a bit hard to follow :) ).

I'm happy to put a PR together to get this sorted.

**Comment by steveworley** [comment]
Agreed, I think it makes sense to standardise on the Symfony2 standards.
